### PR TITLE
Enable write fault injection in db_stress

### PR DESCRIPTION
**Pull request by anand1976**
This PR depends on #11879 . Enable write fault injection for the basic whitebox, blackbox, and cf_consistency modes. For other test modes like multiops_txn, best_efforts_recovery etc., leave it disabled for now until we can do more testing.

### CODE REVIEW

File: tools/db_crashtest.py [hunk 1]
```diff
@@ -155,7 +155,7 @@
         [t * 16384 if t < 3 else 1024 * 1024 * 1024 for t in range(1, 30)]
     ),
     # Sync mode might make test runs slower so running it in a smaller chance
-    "sync": lambda: random.choice([1 if t == 0 else 0 for t in range(0, 20)]),
```
Comment (cbi42) [inline tools/db_crashtest.py:158]: Is this change needed?

File: tools/db_crashtest.py [hunk 2]
```diff
@@ -179,7 +179,7 @@
     "max_key_len": 3,
     "key_len_percent_dist": "1,30,69",
     "read_fault_one_in": lambda: random.choice([0, 32, 1000]),
-    "write_fault_one_in": 0,
+    "write_fault_one_in": lambda: random.choice([0, 32, 1000]),
```
Comment (cbi42) [inline tools/db_crashtest.py:182]: I feel setting it to 32 may not allow a large compaction to finish, is this intended?

**Review by cbi42 — APPROVED** [review]
LGTM

File: tools/db_crashtest.py (hunk 1 above)
Comment (anand1976) [reply tools/db_crashtest.py:158]: Temporary, for testing the write fault injection.

File: tools/db_crashtest.py (hunk 2 above)
Comment (anand1976) [reply tools/db_crashtest.py:182]: Didn't see any issues when running locally, but I had disabled read fault injection. With both enabled, maybe this is too aggressive. I'll increase it and let's see how it goes.
